VKApp/VKApp/Friends/FriendDetailCollectionViewController.swift

```diff
@@ -43,27 +43,23 @@ final class FriendDetailCollectionViewController: UICollectionViewController {
 
     // MARK: - Private Methods
 
-    private func loadDataFromRealm() {
-        do {
-            let realm = try Realm()
-            let friendPhotos = Array(realm.objects(PhotoItem.self))
-            if photos != friendPhotos {
-                photos = friendPhotos
-            } else {
-                fetchUserPhotos()
-            }
-        } catch {
-            showAlertError(title: Constants.titleErrorText, message: error.localizedDescription)
+    private func loadData() {
+        guard let friendPhotos = realmService.loadData(PhotoItem.self) else { return }
+        let friendsPhotos = Array(friendPhotos)
+        if photosItem != friendsPhotos {
+            photosItem = friendsPhotos
+        } else {
+            fetchUserPhotos()
         }
     }
 
     private func fetchUserPhotos() {
-        networkService.fetchUserPhotos(ownerId: friendId) { [weak self] result in
+        NetworkService().fetchUserPhotos(ownerId: friendId) { [weak self] result in
```

Comment:
```suggestion
        networkService().fetchUserPhotos(ownerId: friendId) { [weak self] result in
```
мы создаем инстенс сервиса, чтобы потом прокинуть его в ячейку, для того чтобы сделать запрос на получение фото,
если у тебя несколько типов запросов на одной странице? для каждой кнопки повторно будем создавать экземпляр класса?  верни как было